src/linkplay/bridge.py

```diff
@@ -148,6 +148,10 @@ async def set_play_mode(self, mode: PlayingMode) -> None:
             LinkPlayCommand.SWITCH_MODE.format(PLAY_MODE_SEND_MAP[mode])
         )  # type: ignore[str-format]
 
+    async def play_preset(self, preset_number: int) -> None:
+        """Play a preset."""
+        await self.bridge.request(LinkPlayCommand.PLAY_PRESET.format(preset_number))
```

Comment:
We should check if the `preset_number` falls between 0 and 9 (10?) through assertion. We can then also write tests that validate this.

https://developer.arylic.com/httpapi/#play-preset-content

Reply:
Good call, will add shortly

Reply:
Though from my testing the "frist" preset seems to be 1 (so 1-10 rather than 0-9).  Not sure if all players will have similarly 1-based indexing though